.github/workflows/e2e-tests.yml

```diff
@@ -19,17 +19,24 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Set up Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: '20'
+      # Build the Docker image using the Dockerfile in the ./e2e/ folder
+      - name: Build Docker image
+        run: docker build -t playwright-e2e -f ./e2e/Dockerfile .
```

Comment:
we can wrap the docker build  in a make target like `make e2e-build` and `make e2e-run`